moz_logging/sandboxes/heka/analysis/moz_logging_ingestion_error_monitor.lua

```diff
@@ -175,21 +219,12 @@ function timer_event(ns, shutdown)
         if th and not alert.throttled(k) then
             local e = cb:current_time() - 60e9 -- exclude the current minute
             local s = e - ((MINS_IN_HOUR - 1) * 60e9)
-            local array = cb:get_range(INGESTED, s, e)
-            local isum, vcnt = stats.sum(array)
-            if not inactivity_alert(ns, th, k, vcnt, cb, e) then
-                array = cb:get_range(ERROR, s, e)
-                local esum = stats.sum(array)
-                if isum > 1000 or esum > 1000 then
-                    local pe  = esum / (isum + esum) * 100
-                    if pe > th.percent then
-                        if alert.send(k, "ingestion error",
-                                      string.format(ingestion_error_template, isum, esum, pe,
-                                                    th.percent, alert.get_dashboard_uri(k),
-                                                    diagnostic_dump(diags))) then
-                            cb:annotate(ns, ERROR, "alert", string.format("%.4g%%", pe))
-                        end
-                    end
+            local vcnt = error_alert(ns, th, k, cb, s, e)
+            inactivity_alert(ns, th, k, vcnt, cb, e)
+            if th.latency > 0 and latency > th.latency then
```

Comment:
Now that there's `error_alert` and `inactivity_alert`, it might be nice to factor out `latency_alert` as well.

Reply:
Heh, I had it broken out into a function at one point but it does so little I decided to in-line it